static/app/views/insights/database/components/databaseSystemSelector.spec.tsx

```diff
@@ -219,7 +219,6 @@ describe('DatabaseSystemSelector', () => {
     expect(dropdownOptionLabels[1]).toHaveTextContent('MongoDB');
 
     await userEvent.click(dropdownOptionLabels[0]!);
-    expect(dropdownSelector).toHaveTextContent('SystemPostgreSQL');
```

Comment:
note: for this test, the `useLocalStorage` hook is mocked so we don’t really have state management that would make sure the text updates. We previously relied on the value updating automatically from within. Now, we need state.